src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderSpecialImportsRegExp.java

```diff
@@ -0,0 +1,19 @@
+/*
+CustomImportOrder
+customImportOrderRules = SPECIAL_IMPORTS###STANDARD_JAVA_PACKAGE
+standardPackageRegExp = junit
+thirdPartyPackageRegExp = (default).*
+specialImportsRegExp = (default)^$
+separateLineBetweenGroups = (default)true
+sortImportsInGroupAlphabetically = (default)false
+
+
+*/
+
+//non-compiled with javac: special package and requires imports from the same package
```

Comment:
Remove this comment, ilit is compile able.

Reply:
done